debezium-core/src/main/java/io/debezium/relational/history/AbstractDatabaseHistory.java

```diff
@@ -131,7 +131,19 @@ else if (ddl != null && ddlParser != null) {
                     }
                     try {
                         logger.debug("Applying: {}", ddl);
-                        ddlParser.parse(ddl, schema);
+
+                        try {
+                            ddlParser.parse(ddl, schema);
+                        } catch (RuntimeException parserException) {
+                            final String fixedDdl = ddl.replaceAll("\\s+", " ").trim();
+                            if (!fixedDdl.matches("(?i)^\\w+ TO \\w+$")) {
```

Comment:
@mikekamornikov given the latest connector failures:
```
    Name:         ondemandus07-us-west-2-56647dea69db19b3e0002a71-cdc-source
    Tasks:
      Id:     0
      State:  FAILED
      Trace:  io.debezium.text.ParsingException: DDL statement couldn't be parsed. Please open a Jira issue with the statement '`db_buyshade`.`email_addr_bean_rel-bak`
                  TO `db_buyshade`.`email_addr_bean_rel_bak`'
mismatched input '`db_buyshade`' expecting {<EOF>...
```
It looks like this regex doesn't match a quoted identifier. It could be something like: ``"(?i)^[`\\w]+ TO [`\\w]+$"``.